script/1_0_2current_en_script/Ciel/Day 2/02_01_CIEL02_1_3.txt

```diff
@@ -49,14 +49,14 @@ It was so expensive that it was discontinued within a week, but many students ar
 "Fufufu, well you see..."
 Senpai smiles as if melting with joy, places the paper bag on her lap, and takes out four side dishes from inside.
 <Page17>
-"Ta-da! It's been a week since I advised the school store's stocking routes! At last! They can sell authentic bakery products!"
+"Ta-da! It's been a week since I advised the school store's supply chain! At last! They can sell authentic bakery products!"
 <Page18>
-It is an oval-shaped, chunky loaf of bread that spilled slightly from the palm of her hand.
-It is a deep-fried bread with a dark brown surface that glistened with oil.
+It is an oval-shaped, chunky loaf of bread that spills slightly from the palm of her hand.
+Deep-fried with a dark brown surface that glistens with oil.
 All four are the same size and design.
 No matter how you look at it, it's freshly fried curry bread.
 <Page19>
-"Good...this is as good as freshly baked...at 300 yen per piece, they are rather expensive, but it is no exaggeration to say that they are the pinnacle of student luxury.
+"Good...this is as good as freshly baked...at 300 yen per piece, they are rather expensive, but perhaps it is no exaggeration to say that they are the pinnacle of student luxury. //Maybe too long, a way to improve it?
```

Comment:
A rich, full-bodied curry bun, sensibly priced at 300 yen a piece. Now, for a little magic, I'm going to make this curry bread disappear. 
> Good...This is as good as freshly baked, and at 300 yen a piece, they're pretty expensive. It wouldn't be an exaggeration to say this is the pinnacle of student luxury.